components/button/index.css

```diff
@@ -306,6 +306,10 @@ a.spectrum-Button {
 /* Windows High Contrast Mode */
 @media (forced-colors: active) {
 	.spectrum-Button {
+		--highcontrast-button-content-color-default: ButtonText;
+		--highcontrast-button-content-color-hover: ButtonText;
+		--highcontrast-button-content-color-focus: ButtonText;
+		--highcontrast-button-content-color-down: ButtonText;
```

Comment:
In addition, I think it would be a good idea to be explicit about setting the matching background color too. Right now we're relying on the default.
```
--highcontrast-button-background-color-default: ButtonFace;
--highcontrast-button-background-color-hover: ButtonFace;
--highcontrast-button-background-color-down: ButtonFace;
--highcontrast-button-background-color-focus: ButtonFace;
--highcontrast-button-background-color-disabled: ButtonFace;
```

Reply:
Updated here https://github.com/adobe/spectrum-css/pull/3078/commits/853ddac2d64748b9dadadc5253614ac63b61e0ff#diff-97b5a7e747486aca74b3076354c28a00ef69dab3c91dc0806d926078198571a2R313 ✨